src/Compiler/FSCompCheck.fsx

```diff
@@ -0,0 +1,22 @@
+#r "nuget: FSharpPlus, 1.4.1"
```

Comment:
this might not work in some cases in CI, since in some environments there are no pre-defined nuget sources, so it will only perform search in cache.

Reply:
Thanks for your explanation!